interface/views.py

```diff
@@ -188,6 +185,39 @@ def ProcessRepo(request, full_name):
     return redirect(url)
 
 
+def search_view(request, full_name):
+    query_text = request.GET.get('q', None)
+    if not query_text:
+        raise Http404
+
+    repo = get_object_or_404(Repo, full_name=full_name)
+
+    vector = SearchVector('body')
+    query = SearchQuery(query_text+':*')
+
+    docs = Document.objects.annotate(
```

Comment:
Lets write some tests for this. We can also pull this out and put it in a wrapper, so if we change the way we do search it will have all the logic in one place.